src/node/ccf_acme_client.h

```diff
@@ -31,21 +39,133 @@ namespace ccf
     }
   }
 
+  class ACMERpcEndpoints : public CommonEndpointRegistry
+  {
+  public:
+    ACMERpcEndpoints(
+      NetworkState& network, ccfapp::AbstractNodeContext& context) :
+      CommonEndpointRegistry(get_actor_prefix(ActorsType::well_known), context)
+    {
+      auto handler = [this](auto& ctx) {
+        http_status response_status = HTTP_STATUS_INTERNAL_SERVER_ERROR;
+        std::string response_body;
+
+        try
```

Comment:
This `try ... catch` logic should already be generically handled by the HTTP server frontend.